apps/jobs/migrations/0005_auto_20170211_0732.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you please rename this migration file ?

Reason I am asking this it becomes just by looking at the filename what the migration is about.

You can create migration by running 

```
python manage.py makemigration <name_of_app> --name=<some_suitable_migration_name>
```